.circleci/config.yml

```diff
@@ -206,9 +206,18 @@ jobs:
             yarn build
             BYPASS_AUTH=true yarn start:ci
           background: true
+          environment:
+            POSTGRES_USERNAME: postgres
+            POSTGRES_DB: ttasmarthub
       - run:
           name: Run migrations ci
           command: yarn run db:migrate:ci
+      - run:
+          name: Seed database
+          command: yarn db:seed
+          environment:
+            POSTGRES_USERNAME: postgres
+            POSTGRES_DB: ttasmarthub
```

Comment:
Instead of naming these variables twice, I think you can [use the `environment` key at the job level](https://circleci.com/docs/2.0/env-vars/?utm_medium=SEM&utm_source=gnb&utm_campaign=SEM-gb-DSA-Eng-uscan&utm_content=&utm_term=dynamicSearch-&gclid=Cj0KCQiA2af-BRDzARIsAIVQUOfLm5QJc1iQTMOgJFBztJ7z-GQxZNrk0dAxhx-OwUyFDeglv5J3HDcaAsRlEALw_wcB#setting-an-environment-variable-in-a-job) and thereby supply them once.